src/utils/index.ts

```diff
@@ -13,6 +13,8 @@ export {
   mergeRefs,
   composeEventHandlers,
 };
+export * from './wrapStringChild';
+export * from '../components/primitives/Radio/RadioGroup';
```

Comment:
replace `*` with the intended name.

Reply:
Thank you for your feedback. I fixed it in the recent commit.